stylesheets/_component.rules.scss

```diff
@@ -498,7 +519,7 @@ td.table__form .rules + .form__group:last-of-type {
 
     .rule-block--if,
     .rule-block--else-if {
-        background-color: lighten($rule-color-if, 40);
+        background-color: lighten($rule-color-if, 41);
```

Comment:
These bg color changes are also in develop so why are these showing in the diff?